### PR TITLE
Frontend structure

### PR DESCRIPTION
**Pull request by roeckerc**
front end structure complete as per the trello https://trello.com/c/bhLpFHti/1-frontend-structure

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,46 @@
...
+import CatIndex from './pages/CatIndex'
+import CatShow from './pages/CatShow'
+import CatNew from './pages/CatNew'
+import CatEdit from './pages/CatEdit'
+import Home from './pages/Home'
+import NotFound from './pages/NotFound'
+import { Routes, Route } from "react-router-dom"
+import cats from './mockCats.js'
+// import 'bootstrap/dist/css/bootstrap.min.css';
+// import Navbar from './components/Header'
```
Comment (WalkerTexas-coder) [inline src/App.js]: Let's remove this commented out code from production

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <div className='footer'>
+      <footer >
+        <div className='max-height'>
+          {/* <img src="photos/footer.png" /> */}
+        </div>
+      <p> Created by Luis Laurel & Clifford Roecker © 2022</p>
+      </footer>
+    </div>
+    </>
```
Comment (WalkerTexas-coder) [inline src/components/Footer.js]: Clean up whitespace and indentation

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+        <Route path="/catindex" element={<CatIndex />} />
+        <Route path="/catshow" element={<CatShow />} />
+        <Route path="/catnew" element={<CatNew />} />
+        <Route path="/catedit" element={<CatEdit />} />
+        <Route path="/*" element={<NotFound />} />
+      </Routes>
+
+    <Footer />
+  </>
+
```
Comment (WalkerTexas-coder) [inline src/App.js]: Clean up whitespace

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
Pull your tests for the next card out of this PR and I'll continue to review. Be sure to only do the work you are assigned during each card. Code that is for notes or future work should be held in a different location and not sent up to production for approval.

File: src/App.js (hunk 1 above)
Comment (roeckerc) [reply src/App.js]: removed commented out code from production

File: src/components/Footer.js (hunk 2 above)
Comment (roeckerc) [reply src/components/Footer.js]: whitespace and indentation is fixed

File: src/App.js (hunk 3 above)
Comment (roeckerc) [reply src/App.js]: whitespace has been eliminated

**Review by WalkerTexas-coder — APPROVED** [review]
Much better!

**Pull request merged**